Task 2/Get The Middle Character.js

```diff
@@ -0,0 +1,10 @@
+/*
+Question Link
+http://www.codewars.com/kata/get-the-middle-character
+*/
+function getMiddle(myString) {
+  return myString.length % 2
```

Comment:
Since your using the length of the variable multiple times, store that value in a variable within the scope of the function

```
const strLength = inputString.length;
```